src/ca/noxid/lab/gameinfo/GameInfo.java

```diff
@@ -26,7 +26,7 @@
 	private String imageExtension;
 	private File dataDir;
 	private BlConfig gameConfig;
-	private static final String CSPLUS_IMG_EXT = ".png";
+	private static final String CSPLUS_IMG_EXT = ".bmp";
```

Comment:
you're right on assuming this might break some custom stuff but at this point it's so old as to be irrelevant anyway. I really thought it used PNGs... my mind must be playing tricks on me.